service_notification/src/main/kotlin/org/thechance/service_notification/endpoints/model/UserDto.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.service_notification.endpoints.model
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class UserDto(
+    val id: String,
+    val deviceTokens: List<String>,
+    val topics: List<Int>,
+    val notifications: List<String>
```

Comment:
I'm not sure what you mean by "notification". Do you mean the ID, the content, or the title? I also don't know what you're supposed to save or what this list represents!

Reply:
it's supposed to be ids got it changed.

Reply:
do you mean the naming of notifications should be changed to more explicit one to represent it's notifications ids ? @NadaFeteiha

Reply:
@Shehab-m Please ignore my previous comment, as it is the same as @mosayed01 's. I did not see his comment while I was reviewing the code.

Reply:
no worries thanks for your review.